### PR TITLE
Feat- rating

### PR DESCRIPTION
**Pull request by abhij1607**
image
Rating component with upto 5 stars
check preview - https://deploy-preview-25--sapota-ui-temp.netlify.app/docs/rating.html

**Please Ignore feat- list commit**

### CODE REVIEW

File: docs/list.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  </head>
+  <body>
+    <header class="top-nav box-shadow main-header">
+      <nav class="navigation">
+        <div class="logo-btn-header">
+          <a href="../index.html" class="nav-brand">SAPOTA UI</a>
+          <button class="menu-icon">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
```
Comment (stutisk) [inline docs/list.html:20]: this can be removed i guess

**Review by stutisk — APPROVED** [review]
clean code ! can go forward with merging it

File: docs/list.html (hunk 1 above)
Comment (abhij1607) [reply docs/list.html:20]: No, It is for the menu button in the mobile view

**Review by prachi-sahani — COMMENTED** [review]
clean and easily understandable code
good to merge

**Comment by abhij1607** [comment]
got good reviews. So, merging

**Pull request merged**